package.json

```diff
@@ -61,7 +61,7 @@
     "ember-frost-bunsen": "^16.0.1",
     "ember-frost-fields": "^6.0.1",
     "ember-frost-info-bar": "^10.0.0",
-    "ember-frost-list": "^7.0.0",
+    "ember-frost-list": "7.0.3",
```

Comment:
do not pin please. Also, u should be able to get that version of ember-frost-list with the current versioning (just need to run npm update or do a fresh npm install)

Reply:
@IMFIL https://semver.org has more info about how/why this works the way @EWhite613 outlined if interested.